test/Bridges/Variable/flip_sign.jl

```diff
@@ -24,6 +24,9 @@ config = MOIT.TestConfig()
         vis = MOI.get(bridged_mock, MOI.ListOfVariableIndices())
         y = vis[4]
         @test y.value == -1
+        
+        con = MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorOfVariables, MathOptInterface.Nonpositives}())[1]
```

Comment:
Use `test_models_equal` to test the getter of `ConstraintSet`. See
https://github.com/JuliaOpt/MathOptInterface.jl/blob/5b25a050d99efb1a57a796d4ca2ffc5cdcf44e17/test/Bridges/Constraint/geomean.jl#L29-L72